### PR TITLE
added a PrintButton component that includes a modal.

### PR DESCRIPTION
**Pull request by benadida**
addresses https://github.com/votingworks/vxmail/issues/84

### CODE REVIEW

File: client/src/screens/TestDeckScreen.tsx [hunk 1]
```diff
@@ -37,8 +37,8 @@ const generateTestDeckBallots = ({
   precinctId,
 }: GenerateTestDeckParams) => {
   const precincts: string[] = precinctId
-    ? [precinctId]
-    : election.precincts.map(p => p.id)
+			    ? [precinctId]
+			    : election.precincts.map(p => p.id)
```
Comment (eventualbuddha) [inline client/src/screens/TestDeckScreen.tsx:41]: Battling formatters?

File: client/src/components/PrintButton.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const print = async () => {
+    setIsPrinting(true)
+    setTimeout(() => {
+      setIsPrinting(false)
+    }, 3000)
+    const documentTitle = document.title
+    if (title) {
+      document.title = title
+    }
+    await (window.kiosk ?? window).print()
```
Comment (eventualbuddha) [inline client/src/components/PrintButton.tsx:27]: Will need a more robust error handling strategy eventually.

File: client/src/screens/TestDeckScreen.tsx (hunk 1 above)
Comment (benadida) [reply client/src/screens/TestDeckScreen.tsx:41]: I think we don't have the right hooks, but let me check.

Comment (beausmith) [reply client/src/screens/TestDeckScreen.tsx:41]: I think the issue is that Brian and I rely upon VS Code linting. Ben has been relying upon pre-commit hooks which exist in other repos but not yet in `election-manager`.

File: client/src/components/PrintButton.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  return (
+    <React.Fragment>
+    <Button onPress={print} {...rest}>
+    {children}
+    </Button>
+    <Modal
+      isOpen={isPrinting}
+      centerContent
+      content={
+	<Prose>Printing....</Prose>
```
Comment (beausmith) [inline client/src/components/PrintButton.tsx:42]: Use the `Loading` component here to get an animated ellipsis.

**Pull request merged**